src/subscription_manager/managercli.py

```diff
@@ -2263,15 +2265,15 @@ def _do_command(self):
                             data['management_enabled'] = _("No")
 
                         print columnize(AVAILABLE_SUBS_LIST, _none_wrap,
-                                data['productName'],
-                                data['providedProducts'],
-                                data['productId'],
-                                data['contractNumber'] or "",
+                                self.highlight_by_filter_string(data['productName'], trans_filter),
```

Comment:
Could this be handled by columnize()'s output callback (ie, where _none_wrap is used)?

Maybe needs some info about the output format (ie, tty or not) passed to it in some fashion.

If columnize() or it's callbacks did it, that might be useful so columnize wouldnt try to word wrap it, etc.